strapi/strapi
# [Admin] Replace max-height on Content Types listing

### What does it do?

Sets the `max-height` of the collection types listing in the side bar, to match the full viewport height minus the room taken by the other sections.

### Why is it needed?

The seemingly randomly chosen max height of 180px on the content type listing, obscures much of the custom types created by the user, while there is plenty of room in the side bar.

### How to test it?

- `yarn build`
- `yarn develop`
- See how the content types listing is no longer unnecessarily obscured.

### Related issue(s)/PR(s)

Fixes #10040

<img width="234" alt="image" src="https://user-images.githubusercontent.com/30017/129790029-92b74e83-4e00-40bd-82f4-d05151d6e026.png">
